File: linshi/Product_registration/code/pkgui/unit2.go
// 由res2go IDE插件自动生成，不要编辑。
package pkgui

import (
    "github.com/ying32/govcl/vcl"
    _ "embed"
)

type TForm2 struct {
    *vcl.TForm
    Panel1          *vcl.TPanel
    DateTimePicker1 *vcl.TDateTimePicker
    DateTimePicker2 *vcl.TDateTimePicker
    Label1          *vcl.TLabel
    Label2          *vcl.TLabel
    Panel2          *vcl.TPanel
    Button1         *vcl.TButton
    Button2         *vcl.TButton
    Panel3          *vcl.TPanel
    ProgressBar1    *vcl.TProgressBar
    SaveDialog1     *vcl.TSaveDialog

    //::private::
    TForm2Fields
}

var Form2 *TForm2




// vcl.Application.CreateForm(&Form2)

func NewForm2(owner vcl.IComponent) (root *TForm2)  {
    vcl.CreateResForm(owner, &root)
    return
}

//go:embed resources/unit2.gfm
var form2Bytes []byte

// 注册Form资源  
var _ = vcl.RegisterFormResource(Form2, &form2Bytes)

File: linshi/Product_registration/backup/unit2.pas
unit Unit2;

{$mode objfpc}{$H+}

interface

uses
  Classes, SysUtils, Forms, Controls, Graphics, Dialogs, ExtCtrls, EditBtn,
  DBCtrls, ExtDlgs, StdCtrls, Menus, ComCtrls, DateTimePicker;

type

  { TForm2 }

  TForm2 = class(TForm)
    Button1: TButton;
    Button2: TButton;
    DateTimePicker1: TDateTimePicker;
    DateTimePicker2: TDateTimePicker;
    Label1: TLabel;
    Label2: TLabel;
    Panel1: TPanel;
    Panel2: TPanel;
    Panel3: TPanel;
    ProgressBar1: TProgressBar;
    SaveDialog1: TSaveDialog;
    procedure Button1Click(Sender: TObject);
    procedure Button2Click(Sender: TObject);
  private

  public

  end;

var
  Form2: TForm2;

implementation

{$R *.lfm}

{ TForm2 }

procedure TForm2.Button1Click(Sender: TObject);
begin

end;

procedure TForm2.Button2Click(Sender: TObject);
begin

end;

end.

